website/docs/guides/vector-search.mdx

```diff
@@ -144,7 +144,7 @@ A typical vector search query includes the query vector, the path to the field c
 | `path`         | string   | The path to the field containing the vector.                                                                 | All              |
 | `k`            | integer  | The number of neighbors (results) to return.                                                                 | All              |
 |                |          |                                                                                                              |                  |
-| `efSearch`     | integer  | The number of neighbors or dynamic candidate list to search (default: 40). Must range from 1 to 1000         | HSNW             |
+| `efSearch`     | integer  | The number of neighbors or dynamic candidate list to search (default: 40). Must range from 1 to 1000         | HNSW             |
```

Comment:
These items were odd. I'm not absolutely certain about them, but ... maybe?

Reply:
https://en.wikipedia.org/wiki/Hierarchical_navigable_small_world The fix is correct.